release-notes/breaking-changes/elastic-apm.md

```diff
@@ -17,12 +17,19 @@ To learn how to upgrade, check out <uprade docs>.
 % **Action**<br> Steps for mitigating deprecation impact.
 % ::::
 
-% ## 9.0.0 [elastic-apm-900-breaking-changes]
-% **Release date:** March 25, 2025
+## 9.0.0 [elastic-apm-900-breaking-changes]
+**Release date:** March 25, 2025
 
 % ::::{dropdown} Title of breaking change 
 % Description of the breaking change.
 % For more information, check [PR #](PR link).
 % **Impact**<br> Impact of the breaking change.
 % **Action**<br> Steps for mitigating deprecation impact.
-% ::::
\ No newline at end of file
+% ::::
+
+::::{dropdown} Change server information endpoint / to only accept GET and HEAD requests
+This will surface any agent misconfiguration causing data to be sent to `/` instead of the correct endpoint, e.g. `/v1/traces` for OTLP/HTTP.
```

Comment:
```suggestion
This will surface any agent misconfiguration causing data to be sent to `/` instead of the correct endpoint (for example, `/v1/traces` for OTLP/HTTP).
```